ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol PersonaCellProtocol {
+    @objc var persona: MSFAvatarState? { get set }
```

Comment:
This should be a get only. We create it internally and allow access to the calling code.

Reply:
Removing the get will prevent me from setting the state values in the calling code